frameworks/hdfs/tools/docker-client/krb5.conf

```diff
@@ -0,0 +1,121 @@
+[libdefaults]
+        default_realm = REALM
+
+# The following libdefaults parameters are only for Heimdal Kerberos.
+        v4_instance_resolve = false
+        v4_name_convert = {
+                host = {
+                        rcmd = host
+                        ftp = ftp
+                }
+                plain = {
+                        something = something-else
+                }
+        }
+        fcc-mit-ticketflags = true
+
+[realms]
+        LOCAL = {
+                kdc = KDC_ADDRESS
+                admin_server = KDC_ADDRESS
+        }
+        ATHENA.MIT.EDU = {
```

Comment:
Can we drop all these extraneous realms?

Reply:
Yeah, hoping these aren't required for our intentions for some non-obvious reasons...